CHANGELOG.md

```diff
@@ -52,6 +53,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Major refactoring of GenericSetServices
+  Work is not completed, but a new layer is introduced with the intent that the user SetServices is called
+  from with in the new layer as opposed to the previous mechanism that obligated user SetServices to call
+  generic.   That call is now deprecated.   Significant cleanup remains.
```

Comment:
@mathomp4 I think this must be from a garbled history.  Isn't this only on the v3 branch?

Reply:
Yeah. That shouldn't be on `develop` anymore. It's almost like Weiyuan started before we did the Great Reversion, but GitHub played it "safe" during a merge?

For now he should just remove this entry and push

Reply:
```suggestion.
```
Making it easy for @weiyuan-jiang  - just has to click the button.

Reply:
> 

Exactly, I am puzzled when I tried to resolve the conflict

Reply:
> 

Why I can't see the button?

Reply:
I think it because I used the "suggest" button after this thread was started.  Let me try a new review.